src/Stripe.net/Services/Sources/StripeSourceService.cs

```diff
@@ -40,6 +40,16 @@ public virtual StripeSource Detach(string customerId, string sourceId, StripeReq
             );
         }
 
+        public virtual StripeList<StripeSourceTransaction> SourceTransactions(string sourceId, StripeSourceTransactionsOptions options = null, StripeRequestOptions requestOptions = null)
```

Comment:
I was just looking through existing services and it seems like every one we have now either has its list functionality in a method called `List` or is prefixed with `List` (e.g. `ListLineItems` in `StripeInvoiceService`).

I suspect that we want to either add `List` to the beginning of this method's name, or move it into a separate source transactions service and just call it `List`. Thoughts?

Reply:
Ah, very good point!

Retrieving refunds for an application fee or reversals for a transfer is done via a `List` method on `StripeNestedResourceService`, so we should follow that pattern. I created a new `StripeSourceTransactionService` with a single `List` (+ `ListAsync`) method.